---
title: "Structure and Evolution of Indian Physics Co-authorship Networks"
collection: publications
permalink: /publication/paper1
excerpt: 'This study compares the peak-callers available for ChIP-exo reads using publically available datasets of Glucocorticoid receptors in three cell lines.'
date: 2020-02-21
venue: 'BMC Bioinformatics'
paperurl: 'https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-020-3403-3'

---
Background

ChIP (Chromatin immunoprecipitation)-exo has emerged as an important and versatile improvement over conventional ChIP-seq as it reduces the level of noise, maps the transcription factor (TF) binding location in a very precise manner, upto single base-pair resolution, and enables binding mode prediction. Availability of numerous peak-callers for analyzing ChIP-exo reads has motivated the need to assess their performance and report which tool executes reasonably well for the task.

Results

This study has focussed on comparing peak-callers that report direct binding events with those that report indirect binding events. The effect of strandedness of reads and duplication of data on the performance of peak-callers has been investigated. The number of peaks reported by each peak-caller is compared followed by a comparison of the annotated motifs present in the reported peaks. The significance of peaks is assessed based on the presence of a motif in top peaks. Indirect binding tools have been compared on the basis of their ability to identify annotated motifs and predict mode of protein-DNA interaction.

Conclusion

By studying the output of the peak-callers investigated in this study, it is concluded that the tools that use self-learning algorithms, i.e. the tools that estimate all the essential parameters from the aligned reads, perform better than the algorithms which require formation of peak-pairs. The latest tools that account for indirect binding of TFs appear to be an upgrade over the available tools, as they are able to reveal valuable information about the mode of binding in addition to direct binding. Furthermore, the quality of ChIP-exo reads have important consequences on the output of data analysis.

Recommended citation: **Sharma, V.**, Majumdar, S. Comparative analysis of ChIP-exo peak-callers: impact of data quality, read duplication and binding subtypes. BMC Bioinformatics 21, 65 (2020). https://doi.org/10.1186/s12859-020-3403-3
